.github/workflows/md-link-checker.yml

```
@@ -0,0 +1,14 @@
+name: Check Markdown links
+on:
+  schedule:
+    - cron: '0 13 * * *'
```

```suggestion
    - cron: '0 0 * * 0,4'
```

Alternative `crontab` to run "twice a week (sunday and thursday)".  I personally find that this frequency is better and less annoying.  Totally is personal preference though.  Just thought I would share

---

Yeah, I guess it makes sense. We'll try that :)